lib/fastlane/plugin/firebase_app_distribution/client/aab_info.rb

```diff
@@ -0,0 +1,42 @@
+class AabInfo
+  # AAB states
+  class AabState
+    UNSPECIFIED = "AAB_STATE_UNSPECIFIED"
+    PLAY_ACCOUNT_NOT_LINKED = "PLAY_ACCOUNT_NOT_LINKED"
+    NO_APP_WITH_GIVEN_BUNDLE_ID_IN_PLAY_ACCOUNT = "NO_APP_WITH_GIVEN_BUNDLE_ID_IN_PLAY_ACCOUNT"
+    APP_NOT_PUBLISHED = "APP_NOT_PUBLISHED"
+    PLAY_IAS_TERMS_NOT_ACCEPTED = "PLAY_IAS_TERMS_NOT_ACCEPTED"
+    ACTIVE = "ACTIVE"
+    UNAVAILABLE = "AAB_STATE_UNAVAILABLE"
+  end
+
+  def initialize(response)
+    @response = response || {}
+  end
+
+  def integration_state
+    @response[:integrationState]
+  end
+
+  def test_certificate
+    @response[:testCertificate]
+  end
+
+  def md5_certificate_hash
+    test_certificate ? test_certificate[:hashMd5] : nil
+  end
+
+  def sha1_certificate_hash
+    test_certificate ? test_certificate[:hashSha1] : nil
+  end
+
+  def sha256_certificate_hash
+    test_certificate ? test_certificate[:hashSha256] : nil
+  end
+
+  def empty_certs?
```

Comment:
Possible bug?  All three certs need to be missing for this to evaluate to true.  Shouldn't this be true if _any_ are nil?

And a nit: could this be renamed to `certs_provided?` which evaluates to true when all three are non-nil and non-empty?  This will require you to invert some logic where `empty_certs?` gets used, but will let you get rid of the double-negative `unless` [clause](https://github.com/fastlane/fastlane-plugin-firebase_app_distribution/pull/219/files#diff-1f50a993e035d5ff62885ac3fddac664536c4ca69bb273ea666d890d3b2b5621R42)

Reply:
Nope, the backend either returns all or nothing. But I like `certs_provided?`, will update.

Reply:
Done.

Reply:
Ok but in the surprising event if only one of the three are nil, wouldn't you want to also crash & burn there?